Model/Quote/QuoteAction/Currency.php

```diff
@@ -11,6 +11,7 @@
  */
 class Currency implements QuoteActionInterface
 {
+    private const SET_DISPLAY_CURRENCY = 'SET_DISPLAY_CURRENCY';
```

Comment:
nit: seems the other const are not in capital we might want to follow the same pattern.

Reply:
changed to lowercase